data_structures/queue/queue.py

```diff
@@ -2,29 +2,30 @@ class Queue:
     def __init__(self, size):
         self.items = []
         self.size = size
-        self.number = 0
+        self.count = 0
 
     def is_empty(self):
         return self.items == []
 
     def is_full(self):
-        return self.number == self.size
+        return self.count == self.size
 
     def enqueue(self, data):
-        if not self.is_full():
-            self.items.append(data)
-            self.number +=1
-        else:
+        if self.is_full():
```

Comment:
Use assert here and throw `AssertionError`. [ref](https://openhome.cc/Gossip/CodeData/PythonTutorial/AssertDocTestPy3.html)